### PR TITLE
feat: gateway admin server

### PR DESCRIPTION
**Pull request by scott-dn**
Ref: #119 
Depend on: #107

### CODE REVIEW

File: crates/ursa-gateway/rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+condense_wildcard_suffixes = true
```
Comment (b0xtch) [inline crates/ursa-gateway/rustfmt.toml:1]: is this just formatting for gateway. Should we maintain that across all crates? as we have one in the root folder

**Review by b0xtch — APPROVED** [review]

File: crates/ursa-gateway/rustfmt.toml (hunk 1 above)
Comment (scott-dn) [reply crates/ursa-gateway/rustfmt.toml:1]: Def will do that in next PR, as it’s used nightly and might need more experiments.

Comment (b0xtch) [reply crates/ursa-gateway/rustfmt.toml:1]: i was mainly referring to how the formatting will differ from other crates. We should maintain consistency with other crates

Comment (scott-dn) [reply crates/ursa-gateway/rustfmt.toml:1]: Oh, that is mainly the import formats.
But yeah, I agree we should maintain all crates.

**Review by b0xtch — APPROVED** [review]

**Pull request merged**